source/core/src/main/com/csse3200/game/services/DialogueBoxService.java

```diff
@@ -85,37 +85,32 @@ public void hideCurrentOverlay() {
      * Update the current chat overlay if it exists also responsible for unhighlighting an entity if it exists.
      */
     public void updateText(String[][] text, int priority) {
-        hints = text;
-        if (priority <= curPriority) {
-            curPriority = priority;
-            if (currentOverlay == null) {
-                // handling if it ever gets deleted when not supposed to
-                currentOverlay = new DialogueBox(hints);
-            } else {
-                updateTextHelper();
-            }
-        }
-        if (currentEntity != null) {
-            AnimationRenderComponent animator =  currentEntity.getComponent(AnimationRenderComponent.class);
-            if (animator != null) {
-                animator.startAnimation("float");
-            }
-            currentEntity = null;
-        }
-    }
+        updateTextHelper(text, priority);
+        if (currentEntity != null) currentEntity = null; // if probably isn't necessary, but that is how it
```

Comment:
Is SonarLint/Cloud happy with not having braces here?

Reply:
My sonarlint (which is synced with our project's sonarcloud) has no issue with it.